api/api_test.go

```diff
@@ -251,7 +251,7 @@ func TestServerReadOnlyRoutes(t *testing.T) {
 		{"POST", "/registry/profile/new", 403},
 		{"POST", "/registry/profile/prove", 403},
 		{"GET", "/checkout/", 403},
-		{"GET", "/status/", 403},
+		// {"GET", "/status/", 403},
```

Comment:
This should work with `{"GET", "/status", 403},` ie remove the trailing `/` as gorilla is strict about slashes by default.